buildSrc/src/main/groovy/com/uber/okbuck/generator/BuckFileGenerator.groovy

```diff
@@ -180,11 +180,19 @@ final class BuckFileGenerator {
 
         // Test
         if (target.robolectric && target.test.sources && !target.isTest) {
+            String manifestRuleName = null
+            if (target instanceof AndroidAppTarget) {
+                Rule manifestRule = AndroidManifestRuleComposer.compose(target)
```

Comment:
this is quite heavy.

If we just need the name, its literally just `manifest_${target.name}`

Use the `manifest` static method on any target to get this. There is no need to make another rule